development-vm/replication/sync-aws-mongo.sh

```diff
@@ -31,8 +31,15 @@ status "Starting MongoDB replication from S3: ${SRC_HOSTNAME}"
 if ! $SKIP_DOWNLOAD; then
   mkdir -p $MONGO_DIR
 
-  status "Downloading latest MongoDB backup from S3: ${SRC_HOSTNAME}"
-  aws s3 sync s3://govuk-integration-database-backups/mongo/daily/${SRC_HOSTNAME}/mongodump-$(date '+%Y-%m-%d_%H%M').tgz $MONGO_DIR/
+  # get the last file name from s3
+  remote_file_details=$(aws s3 ls s3://govuk-integration-database-backups/mongodb/daily/${SRC_HOSTNAME}/ | tail -n1)
```

Comment:
Just to check - are these files are named so that the last file is the latest one?

Reply:
yes